easybuild/easyconfigs/c/Cufflinks/Cufflinks-2.2.1-goolf-1.7.20.eb

```diff
@@ -0,0 +1,28 @@
+name = 'Cufflinks'
+version = '2.2.1'
+
+homepage = 'http://cufflinks.cbcb.umd.edu/'
```

Comment:
Yeah, this is the homepage that has been updated in the newer foss and intel toolchains. It should probably be updated here too.